src/stash-list/folder.vue

```diff
@@ -127,6 +173,11 @@ export default defineComponent({
             return getDefaultFolderNameISODate(this.folder.title) !== null
                 ? undefined : this.folder.title;
         },
+        tooltip(): string {
+            const st = this.childTabStats;
+            const statstip = `(stashed: ${this.validChildren.length}, tabs: ${st.open}/${st.discarded}/${st.hidden} open/unloaded/hidden)`;
```

Comment:
I think natural language would be a little easier for users to read, something like:

`(N stashed tabs; X open, Y unloaded, Z hidden)`

Reply:
Makes sense!